swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_eus_payg.yaml

```diff
@@ -0,0 +1,28 @@
+---
+# Cleanup this file in SWATCH-1690 with accurate and more information
+platform: RHEL
+
+id: rhel-for-x86-eus-payg
```

Comment:
if this file in this pull request on purpose? I don't see the relation between the PR and the JIRA ticket, to this new profile.

Reply:
This product configuration is for cost. There will be some placeholders until we receive information from outside services, but SWATCH-1690 will have some more correct information. Its one step ahead so, QE can test because we want to test against actual product rather than BASILISK test product.

Reply:
But what is the point of adding part of SWATCH-1690 here? :/

Reply:
We don't want to utilize BASILISK product for testing here, for that reason. You can see that there are comments with description related to SWATCH-1690 and  PLACEHOLDERS. We are not adding SWATCH-1690 here.

Reply:
@Sgitario you can also check my test steps include `RHEL System` as `service_type` which is being sent by COST. For that reason we have this product for testing purposes.

Reply:
I still don't get why we have partial solutions for other tickets like SWATCH-1690 in unrelated pull requests. 
But if @kahowell is ok with this, go for it.

Reply:
Not that anyone asked my opinion, but I'm fine with this product config file getting added here with placeholders.  We frequently keep cards around that end up being "verify we did this thing totally" because some of it gets done as part of something.

In general, writing stories that are truly standalone and testable is definitely an area we have an opportunity to improve upon.

Reply:
> Not that anyone asked my opinion, but I'm fine with this product config file getting added here with placeholders. We frequently keep cards around that end up being "verify we did this thing totally" because some of it gets done as part of something.

I would not say this is a good practice, but _whoever is without sin, cast the first stone_. :D

> In general, writing stories that are truly standalone and testable is definitely an area we have an opportunity to improve upon.

Of course!